src/com/lms/customExceptions/TransactionException.java

```diff
@@ -0,0 +1,9 @@
+package com.lms.customExceptions;
+
+@SuppressWarnings("serial")
+public class TransactionException extends Exception {
+
+	public TransactionException(String errorMessage) {
```

Comment:
Will we ever want to create an instance that isn't one of the subclasses? If not, make this constructor `protected` instead of `public`.

Reply:
I will actually do as your other comment suggests and just call the super class instead.

Reply:
The two changes are unrelated to each other. But do as you will.

Reply:
I will change that. You are correct.